Swift/BRCrypto/BRCryptoWalletManager.swift

```diff
@@ -109,17 +109,7 @@ public final class WalletManager: Equatable {
     }
 
     internal var height: UInt64 {
-        get { return network.height }
-        set {
-            network.height = newValue
-            announceEvent (WalletManagerEvent.blockUpdated(height: newValue))
-            wallets.flatMap { $0.transfers }
-                .forEach {
-                    if let confirmations = $0.confirmationsAt (blockHeight: newValue) {
-                        $0.announceEvent (TransferEvent.confirmation (count: confirmations))
-                    }
-            }
-        }
+        return network.height
```

Comment:
How is the TransferEvent.confirmation generated now?  Maybe in `walletManagerEventCallback`?

Reply:
Confirmation is captured via CRYPTO_TRANSFER_EVENT_CONFIRMED and a corresponding state.

You have a comment to the effect that is CRYPTO_TRANSFER_EVENT_CONFIRMED is unneeded.

Reply:
Nothing is generating `CRYPTO_TRANSFER_EVENT_CONFIRMED` (it seems?) and the listener callback is stubbed out (with the `10` below):
```
                case CRYPTO_TRANSFER_EVENT_CONFIRMED:
                    transfer.listener?.handleTransferEvent (system: manager.system,
                                                            manager: manager,
                                                            wallet: wallet,
                                                            transfer: transfer,
                                                            event: TransferEvent.confirmation(count: 10))
```

Maybe `BRCryptoTransferState` needs a `u.confirmed.height` field and Crypto C generates 'CONFIRMED' as needed (specially when the blockHeight changes)?